### PR TITLE
Message post

### PR DESCRIPTION
**Pull request by aryaman-titan**

### CODE REVIEW

File: NewConnect.php [hunk 1]
```diff
@@ -9,82 +9,241 @@
...
+include('dbconfig.php');
+$sql = "SELECT userID, name, email, bhawan, username, img, bio, branch_y, follow_count FROM userData ORDER BY follow_count DESC";
 $result = mysqli_query($conn, $sql);
+function hobbieID_to_name($a){
+  if($a==1){return "Dancing";}
+  if($a==2){return "Listening Music";}
+  if($a==3){return "Cricket";}
+  if($a==4){return "Singing";}
+  if($a==5){return "Fooseball";}
+  if($a==6){return "Reading Novels";}
```
Comment (leshnabalara) [inline NewConnect.php]: why is this hardcoded here , can you find a better way to do that

File: ToDo [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+----- >>>> DONE #9 Major reframing of login/signup form validation
+                     ---> Not resetting the whole form on one error
+#10 Setting up separate chatrooms per hobby and feature of mentioning someone
+----- >>>> DONE #11 Displaying users who are online
+----- >>>> DONE #12 Displaying green button if online
+----- >>>> DONE #13 Completely revamping of Login and signup form.
+----- >>>> DONE #14 Designing the forms for better UI/UX
+----- >>>> DONE #15 Polishing of Login/Sigup backend
+#16 Logo Change
+
```
Comment (leshnabalara) [inline ToDo:33]: add this file in .gitignore

File: chat_box.css [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    cursor: pointer;
+}
+.contacts{
+    list-style: none;
+    padding: 0;
+}
+.contacts li{
+    width: 100% !important;
+    padding: 5px 10px;
+    margin-bottom: 15px !important;
```
Comment (leshnabalara) [inline chat_box.css:86]: have a look at this https://css-tricks.com/when-using-important-is-the-right-choice/

File: ToDo (hunk 2 above)
Comment (aryaman-titan) [reply ToDo:33]: Okay! But do I need to clone the repo again or shall I just add 'ToDo' to .gitignore?

File: list.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
+
+var hobbie_names = ["Dancing", "Listening Music", "Cricket", "Singing", "Fooseball", "Reading Novels"];
+var goal_names = ["Gymming", "Tech Group", "UPSC", "Branch Change", "Internship"];
+
+function n_2_i_h(a){
+  var res = hobbie_names.indexOf(a);
+  return res;
+}
+function n_2_i_g(a){
+  var res = goal_names.indexOf(a);
```
Comment (leshnabalara) [inline list.js:10]: use better function names

Comment (aryaman-titan) [reply list.js:10]: Function name changed to name_to_index_hobbie

File: NewConnect.php (hunk 1 above)
Comment (aryaman-titan) [reply NewConnect.php]: Yeah, imo using arrays for the task would be better. I'll just update the code

File: server.php [hunk 5]
```diff
@@ -14,22 +17,35 @@
   $all_users = mysqli_query($db, "SELECT * FROM userData;");
```
Comment (leshnabalara) [inline server.php]: why not use count() directly?

**Review by leshnabalara — COMMENTED** [review]
I have left some comments check them out

File: ToDo (hunk 2 above)
Comment (leshnabalara) [reply ToDo:33]: ni sirf add karne se file track karna band kar dega,phir github se hata dena isse

File: server.php (hunk 5 above)
Comment (aryaman-titan) [reply server.php]: Code updated!

File: NewConnect.php (hunk 1 above)
Comment (aryaman-titan) [reply NewConnect.php]: Changed the code to array format

**Pull request merged**